server/controllers/request.js

```diff
@@ -0,0 +1,105 @@
+const Request = require("../models/Request");
+const asyncHandler = require("express-async-handler");
+
+// @route GET /requests
+// @desc load requests for logged in user
+// @access Private
+exports.loadRequests = asyncHandler(async (req, res, next) => {
+  const requests = await Request.find({ userId: req.user.id });
+
+  if (!requests) {
+    res.status(404);
+    throw new Error("No requests found");
+  }
+
+  res.status(200).json({
+    success: {
+      requests: requests,
+    },
+  });
+});
+
+// @route POST /requests/create
+// @desc create a new request
+// @access Private
+exports.createRequest = asyncHandler(async (req, res, next) => {
+  const { sitterId, start, end } = req.body;
```

Comment:
Optional: We can also validate the request body params and return a `400` if they're missing (ideally we would want to use a middleware to do so - express validator should be a part of the pkg bundle)